dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -184,7 +187,9 @@ Future<TaskResult> runWebBenchmark(WebBenchmarkOptions benchmarkOptions) async {
         }
 
         final String webRendererName;
-        if (benchmarkOptions.useWasm) {
+        if (benchmarkOptions.useDdc) {
+          webRendererName = 'ddc';
```

Comment:
The renderer is still canvaskit. I don't think we need to change this if block.

Reply:
updated